src/stronghold.hs

```diff
@@ -237,13 +237,14 @@ site runStoreOp =
       hier <- Trees.lastHierarchy hist
       snd <$> Trees.materializedView path hier
 
-    let etag = (Base16.encode . hash . toStrict . Aeson.encode) json
+    let jsonBS = (toStrict . Aeson.encode) json
+    let etag = (Base16.encode . hash) jsonBS
```

Comment:
though you could even make them look more like pipelines with `f . g . h $ val`, though it might just be my brain that turns them into nice looking pipelines